store_db.sql

```diff
@@ -0,0 +1,31 @@
+CREATE TABLE book_store (
+      id     INTEGER PRIMARY KEY,
+      title  TEXT,
+      author TEXT,
+      price  NUMERIC,
+      rating NUMERIC);
+
+INSERT INTO book_store (id, title, author, price, rating)
```

Comment:
Each one of these insert statements needs to be separated. I received an error when I imported your file. The following revision will correctly insert into your book_store database:

```INSERT INTO book_store (id, title, author, price, rating) VALUES (1, 'The Handmaids Tale', 'Margaret Atwood', 8.69, 4.05);```